packages/rich-text/src/get-selection-start.js

```diff
@@ -1,12 +0,0 @@
-/**
- * Gets the start index of the current selection, or returns `undefined` if no
- * selection exists. The selection starts right before the character at this
- * index.
- *
- * @param {Object} value Value to get the selection from.
- *
- * @return {?number} Index where the selection starts.
- */
-export function getSelectionStart( { start } ) {
```

Comment:
There's still a reference to this function:

https://github.com/WordPress/gutenberg/blob/0793065c26e76792168af647bff41dc6d34ee862/packages/editor/src/components/rich-text/patterns.js#L9

Reply:
I just fixed it :)